packages/components/src/components/Table/TableRow.tsx

```diff
@@ -1,17 +1,55 @@
 import { ReactNode } from 'react';
 import styled, { css } from 'styled-components';
 
-import { Elevation, mapElevationToBorder } from '@trezor/theme';
+import {
+    Elevation,
+    mapElevationToBackground,
+    mapElevationToBorder,
+    nextElevation,
+} from '@trezor/theme';
 
 import { useElevation } from '../ElevationContext/ElevationContext';
+import { useTable } from './Table';
+import { useTableHeader } from './TableHeader';
 
-export const Row = styled.tr<{ $elevation: Elevation; $isCollapsed: boolean }>`
+export const Row = styled.tr<{
+    $elevation: Elevation;
+    $isCollapsed: boolean;
+    $highlight: boolean;
+    $isHeader: boolean;
+}>`
     border-top: 1px solid ${mapElevationToBorder};
 
     &:first-child {
         border-top: 0;
     }
 
+    ${({ $highlight, theme, $elevation, $isHeader, onClick }) =>
+        $highlight &&
+        !$isHeader &&
+        onClick &&
+        css`
+            &:hover {
+                cursor: pointer;
+                background-color: ${mapElevationToBackground({
+                    theme,
+                    $elevation: nextElevation[$elevation],
+                })};
+
+                & > td:first-child {
+                    background: linear-gradient(
```

Comment:
Perhaps make a const or func out of this so you can use it here and in the Cell? I'll try to figure out if there's a better way to do this, but I think it's good enough for now:)